config.toml.example

```diff
@@ -9,6 +9,12 @@
 # a custom configuration file can also be specified with `--config` to the build
 # system.
 
+# This keeps track of the latest version of `x.py` you've used.
```

Comment:
```suggestion
# Keeps track of the latest version of `x.py` used.
```

Reply:
I don't think this is a good description .. the changes would happen with or without this line, this just makes you aware of them.

Reply:
Okay, maybe I don't put the reason.
```suggestion
# Keeps track of the latest version of `x.py` used.
```
Removes redundant `This` (it couldn't be `that` right?) and `you've` (`you` may not be good and there is only one person here).